apps/ui/src/views/Proposal.vue

```diff
@@ -320,6 +320,9 @@ watchEffect(() => {
               :decimals="votingPowerDecimals"
             />
           </div>
+          <div v-if="space.labels?.length && proposal.labels?.length">
```

Comment:
The wrapping div may be redundant, and we could apply the `if` directly on `ProposalLabels`